src/hooks/audioSdk.ts

```diff
@@ -60,8 +64,13 @@ export function useAudioSdk(
       client.on(EVENT_STREAM_UNPUBLISHED, handleStreamUnpublished);
 
       const token = await fetchSdkToken();
+      await client.join(AGORA_APP_ID, name, token, username);
+      /* const TEST_APP_ID = "d01a910d66284256ab571919c74d14f9";
+      // this needs to be regenerated every 24h
+      const testToken =
+        "006d01a910d66284256ab571919c74d14f9IACBMOPVYegNfhXBXHqp0v08edXY24yhcBZZXkGW60ps81dxXwcAAAAAEABK0cUi6UrxYAEAAQDoSvFg";
```

Comment:
Remove your private test API keys hahah if not automated github scrapers will pick it up and possibly abuse them